polar-core/src/error.rs

```diff
@@ -3,6 +3,9 @@ use std::fmt;
 use indoc::formatdoc;
 use serde::{Deserialize, Serialize};
 
+use crate::resource_block::{Declaration, ShorthandRule};
+use crate::terms::ToPolarString;
```

Comment:
should go into the `use super::{` block below

Reply:
👍🏻